polaris-migrator/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/polaris-migrator",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

```suggestion
  "version": "0.8.0",
```

No need to manually version, the changeset will handle this on the next release 🚢